roles/deploy_jhub/templates/cloud-provider-openstack/cloud-controller-manager-role-bindings.j2

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
```

Comment:
We should have a single comment at the top of these files with a link to the repo they came from for future readers and correct attribution